Src/IronPython/Runtime/PythonContext.cs

```diff
@@ -1887,7 +1887,11 @@ public string GetDefaultEncodingName()
         internal Dictionary<string, object> ErrorHandlers {
             get {
                 if (_errorHandlers == null) {
+#if FEATURE_ENCODING
+                    _errorHandlers = StringOps.CodecsInfo.ErrorHandlers.Value;
```

Comment:
`CodecsInfo` is static which means that this dict would apply to all contexts in an app which is likely undesirable. I would be fine with having something like `Dictionary<string, object> CodecsInfo.CreateErrorHandlers()` and using that (as long as it returned a new dictionary on every call).

Reply:
It is undesirable indeed; I forgot about the contexts.